build.gradle

```
@@ -1,67 +1,80 @@
-            "com.nimbusds:oauth2-oidc-sdk:9.3.1",
-            "com.fasterxml.jackson.core:jackson-core:2.13.0",
-            "com.fasterxml.jackson.core:jackson-databind:2.13.0",
-            "com.fasterxml.jackson.core:jackson-annotations:2.13.0",
-            "org.apache.httpcomponents:httpclient:4.5.13",
-            "org.slf4j:slf4j-simple:1.7.32",
-            "software.amazon.awssdk:dynamodb-enhanced:2.17.89"
+	implementation "com.amazonaws:aws-lambda-java-core:1.2.1",
+			"com.amazonaws:aws-lambda-java-events:3.10.0",
+			"com.amazonaws:aws-java-sdk-dynamodb:1.12.118",
+			"com.nimbusds:oauth2-oidc-sdk:9.3.1",
+			"com.fasterxml.jackson.core:jackson-core:2.13.0",
+			"com.fasterxml.jackson.core:jackson-databind:2.13.0",
+			"com.fasterxml.jackson.core:jackson-annotations:2.13.0",
+			"org.apache.httpcomponents:httpclient:4.5.13",
+			"org.slf4j:slf4j-simple:1.7.32",
+			"software.amazon.awssdk:dynamodb-enhanced:2.17.89"
 
 
-    compileOnly 'org.projectlombok:lombok:1.18.22'
-    annotationProcessor 'org.projectlombok:lombok:1.18.22'
+	compileOnly 'org.projectlombok:lombok:1.18.22'
+	annotationProcessor 'org.projectlombok:lombok:1.18.22'
 
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1',
-            "org.mockito:mockito-core:3.12.4",
-            "org.mockito:mockito-junit-jupiter:4.1.0",
-            "com.github.tomakehurst:wiremock-jre8:2.31.0"
+	testImplementation 'org.junit.jupiter:junit-jupiter:5.8.2',
+			"org.mockito:mockito-core:4.1.0",
+			"org.mockito:mockito-junit-jupiter:4.1.0",
+			"com.github.tomakehurst:wiremock-jre8:2.31.0"
 }
 
-test {
-    useJUnitPlatform ()
+java {
+	sourceCompatibility = JavaVersion.VERSION_11
+	targetCompatibility = JavaVersion.VERSION_11
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
+task buildZip(type: Zip) {
+	from compileJava
+	from processResources
+	into("lib") {
+		from configurations.runtimeClasspath
+	}
 }
 
+test {
+	useJUnitPlatform ()
+	finalizedBy jacocoTestReport
+}
 
-task buildZip(type: Zip) {
-    from compileJava
-    from processResources
-    into("lib") {
-        from configurations.runtimeClasspath
-    }
+jacocoTestReport {
+	dependsOn test
+	reports {
+		xml.enabled true
+	}
 }
 
 sonarqube {
-    properties {
-        property "sonar.projectKey", "alphagov_di-ipv-core-back"
-        property "sonar.organization", "alphagov"
-        property "sonar.host.url", "https://sonarcloud.io"
-    }
+	properties {
+		property "sonar.projectKey", "alphagov_di-ipv-core-back"
+		property "sonar.organization", "alphagov"
+		property "sonar.host.url", "https://sonarcloud.io"
+	}
 }
 
-jacocoTestReport {
-    reports {
-        xml.enabled true
-    }
+spotless {
+	java {
+		target "src/**/*.java"
+		googleJavaFormat("1.13.0").aosp()
+		importOrder "", "javax", "java", "\\#"
+		endWithNewline()
+	}
+	groovyGradle {
+		target '*.gradle' // default target of groovyGradle
+	}
```

I would recommend at least:

```suggestion
  groovyGradle {
    target '**/*.gradle'
    trimTrailingWhitespace()
    endWithNewline()
  }
```

But probably:

```suggestion
  groovyGradle {
    target '**/*.gradle'
    greclipse()
    trimTrailingWhitespace()
    endWithNewline()
  }
```

---

Amended to add groovyGradle